File: CarteiraVacinaDigital.Model/Entities/Employee.cs
using CarteiraVacinaDigital.Model.Entities.Enums;
using Newtonsoft.Json;

namespace CarteiraVacinaDigital.Model.Entities
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Cpf { get; set; }
        public string Coren { get; set; }
        public string Adress { get; set; }
        public string City { get; set; }
        public StateEnum State { get; set; }
        public string Cep { get; set; }
        public OfficeEnum Office { get; set; }
    }
}

File: CarteiraVacinaDigital.Api/ClientApp/src/app/core/model/employee.ts
import { State } from './enum/stateEnum';
import { Office } from './enum/office';

export class Employee {
  id: number;
  name: string;
  email: string;
  password: string;
  cpf: string;
  telefone: string;
  adress: string;
  district: string;
  city: string;
  state: State;
  cep:string;
  coren: string;
  office: Office;
  officeName: string;
}
